_posts/2017-03-15-Performance-of-LSTM-Using-CuDNN-v5.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Performance of LSTM with or without cuDNN(v5) in Chainer
+layout: post
+categories: General
+---
+
+We compare the performance of LSTM with or without cuDNN in Chainer.
+The NVIDIA CUDA® Deep Neural Network library (cuDNN) is a GPU-accelerated library of primitives for deep neural networks.
+cuDNN provides highly tuned implementations for standard routines such as LSTM, CNN.
+
+In this article, we compare the performance of LSTM with or without cuDNN.
+An LSTM (NStepLSTM) implementation is included in Chainer, which can be found here.
```

Comment:
Thank you for your comments!

> Is the NStepLSTM implementation configurable to run with or without cuDNN? 
Yes.  NStepLSTM have the option `use_cudnn` which decide whether call NVIDIA's cuDNN.
https://github.com/pfnet/chainer/blob/master/chainer/functions/connection/n_step_lstm.py#L373

Could you tell me how to describe the difference in English?